app/helpers/application_helper.rb

```diff
@@ -1,2 +1,24 @@
 module ApplicationHelper
+  def bootstrap_class_for(flash_class)
+    { success: "alert-success",
+      error: "alert-danger",
+      alert: "alert-warning",
+      notice: "alert-info" }[flash_class.to_sym] || flash_class.to_s
+  end
+
+  def flash_messages(opts = {})
```

Comment:
Unused method argument - `opts`. If it's necessary, use `_` or `_opts` as an argument name to indicate that it won't be used. You can also write as `flash_messages(*)` if you want the method to accept any arguments but don't care about them.